manifests/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -326,18 +317,6 @@ spec:
       - name: ingressgatewaysdsudspath
         emptyDir: {}
       {{- end }}
-      {{- if .Values.global.sds.enabled }}
-      - name: sdsudspath
-        hostPath:
-          path: /var/run/sds
-      - name: istio-token
```

Comment:
Same here.

Reply:
This one is fine: https://github.com/istio/istio/blob/538c9ab7d7910075c7f030c7e013b4f9acf98f30/manifests/gateways/istio-ingress/templates/deployment.yaml#L307

Reply:
Got it. Thanks.